src/pages/user/components/Lobby.jsx

```diff
@@ -34,16 +39,53 @@ export default function Lobby() {
       .catch((error) => {
         console.error(error);
       });
-  }, []);
+  }, [token]);
+
+  const handleJoinGame = (gameId) => {
+    const fetchOptions = {
+      method: "PUT",
+      headers: {
+        authorization: token,
+      },
+    };
+
+    fetch(`${apiUrl}/games/${gameId}/join`, fetchOptions)
+      .then((res) => {
+        if (res.status === 401) {
+          throw Error("Not Authorized");
+        } else if (res.status !== 200) {
+          throw Error("[500 ERROR] Internal Server Error");
+        }
+        navigate(`/game/${gameId}`);
+      })
+      .catch((error) => {
+        console.error(error);
+      });
+  };
```

Comment:
Are we updating a game or are we creating a new record in `UsersOnGames` table?

Should it be "POST" or "PATCH/PUT", hmmmmm.... What do you think?

Reply:
I'm doing both: updating `Game` table by changing `gameStatus` from "waiting" to "in-progress" AND connecting new player/user(creating a new record in `UsersOnGames` table) to the `Game` table.
So in this case I think it would be PUT/PATCH...

Reply:
Alrighty then, go for it!